app/src/main/java/com/lab/keyneez/presentation/main/like/LikeFragment.kt

```diff
@@ -7,8 +7,17 @@ import com.lab.keyneez.databinding.FragmentLikeBinding
 import com.lab.keyneez.util.binding.BindingFragment
 
 class LikeFragment : BindingFragment<FragmentLikeBinding>(R.layout.fragment_like) {
+    private var _binding: FragmentLikeBinding? = null
+        get() = requireNotNull(_binding) { "binding value is null." }
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        // 여기서 코드 작성
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
     }
```

Comment:
```suggestion
class LikeFragment : BindingFragment<FragmentLikeBinding>(R.layout.fragment_like) {
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        // 여기서 코드 작성
    }
    
    companion object {
        fun newInstance(): LikeFragment {
            return LikeFragment()
        }
    }
}
```
세미나에서 프래그먼트에 newInstance() 함수를 만들어두는 게 좋다고 언급했었어요!

Reply:
넵 참고하도록 하겠습니당!